addon/manifests/chart/templates/service.yaml

```diff
@@ -0,0 +1,29 @@
+---
+{{- if .Values.prometheus.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "controller.fullname" . }}
```

Comment:
`controller.fullname` doesn't match the template functions defined in `./_helpers.tpl`.  The defined template is called `search.fullname`. You need to make these match.

It's just a name, so we can pick either one. I think I prefer `controller.fullname` because its more generic.
There's also `controller.chart` and `controller.name`. Let's keep al in sync.